pom.xml

```diff
@@ -41,7 +41,7 @@
 		<dependency>
 			<groupId>org.projectlombok</groupId>
 			<artifactId>lombok</artifactId>
-			<optional>true</optional>
+			<version>1.18.8</version>
```

Comment:
move version to properties section and inject version value as a parameter (https://stackoverflow.com/questions/37358318/maven-dependency-version-as-property)